packages/fxa-content-server/app/scripts/templates/complete_reset_password.mustache

```diff
@@ -69,7 +69,9 @@
 
       <div class="button-row ">
           <button type="submit">{{#t}}Reset password{{/t}}</button>
-          <button class="button-link remember-password">{{#t}}Remember password? Sign in{{/t}}</button>
+      </div>
+      <div class="links centered">
+        <a href="#" class="remember-password">{{#t}}Remember password? Sign in{{/t}}</a>
```

Comment:
Hmm, at least this one has clickable whitespace on the left/right side. I think the `flex-direction: column` on `.links.centered` is causing that here. "Lost recovery key?" as well.

<img width="622" alt="image" src="https://user-images.githubusercontent.com/13018240/88866676-03c78980-d1d1-11ea-97d0-9e96a0ffdc3c.png">

<img width="628" alt="image" src="https://user-images.githubusercontent.com/13018240/88866844-72a4e280-d1d1-11ea-84b2-c11fa843c784.png">

Reply:
It seems to have two solutions for this issue:
1 - Remove the `.centered`.
2 - On _custom-rows.css line 28, add
```
a {
  margin: auto;
}
```
What solution should I choose?

Reply:
I removed `centered` from a couple of these and that does seem to do the trick. Removing that class when we don't need it seems preferable to adding additional styles.👍